### ISSUE TITLE
POST to election/vote/[id]/edit causes exception

### ISSUE THREAD

**Issue by Unheppcat** [Unheppcat/nor-seasonal-watch-list#19]
/my/vote (or in general) server config: uncaught exception (I guess) on the server side sends back a stacktrace / debug screen to the browser. Got one by doing:

curl -X POST https://secret-plateau-20333.herokuapp.com/election/vote/116/edit

Reported by entotto

**Comment by entotto** [comment]
The stacktrace showed that the exception was thrown here:
https://github.com/Unheppcat/nor-seasonal-watch-list/blob/6028cd0bacc6ceb10a946970cb7ec84aa657d48f/application/src/Controller/ElectionVoteController.php#L72

which I think is as expected*, given that my `curl` command didn't send any payload. What I was concerned was that the debug screen was sent back when there was an exception thrown.

*But also, I suppose `^/election` paths should be gated by the user role here:
 
https://github.com/Unheppcat/nor-seasonal-watch-list/blob/6028cd0bacc6ceb10a946970cb7ec84aa657d48f/application/config/packages/security.yaml#L39-L44

**Comment by Unheppcat** [comment]
Fixed the stacktrace display problem now that the environment and trusted proxies are set correctly. Moved the firewall missing paths into a new ticket.
